parsel/selector.py

```diff
@@ -11,6 +11,16 @@
 from .csstranslator import HTMLTranslator, GenericTranslator
 
 
+class CannotRemovePseudoElement(Exception):
+
+    pass
```

Comment:
nitpick: could you please remove blank lines before pass?

Reply:
done